modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/network/command/WriteIntentSwitchCommand.java

```diff
@@ -26,7 +25,7 @@
 /**
  * State machine command to cleanup on a transaction commit.
  */
-@Transferable(PartitionReplicationMessageGroup.Commands.WRITE_INTENT_SWITCH)
+@Transferable(PartitionReplicationMessageGroup.Commands.WRITE_INTENT_SWITCH_V1)
```

Comment:
I suggest writing in the class description that this class is now obsolete and is only needed for backward compatibility. And specify which one to use now.

Reply:
Sure, I'll do it